### PR TITLE
Filter eauth PII from sentry

### PR DESCRIPTION
**Pull request by annaswims**
## Description of change
Filter PII attributes related to sso before they are sent to sentry

Currently, all fields starting with `va_eauth_` are being filtered in sentry, but this change will allow us to see the non-pii fields and filter before the pii is sent to sentry.

## Testing
specs
## Acceptance Criteria 
@patrickvinograd or @ericbuckley, please take a look at the spec to confirm that the remaining fields are not PII

#### Unique to this PR
After this PR is merged remove `va_eauth_` from `
Additional field names to match against when scrubbing data. Separate multiple entries with a newline
va_eauth_
`  http://sentry.vfs.va.gov/settings/vets-gov/projects/platform-api-staging/
#### Applies to all PRs

- [x] Swagger docs have been updated, if applicable
- [ ] Provide link to originating GitHub issue, or connected to it via ZenHub
- [x] Does not contain any sensitive information (i.e. PII/credentials/internal URLs/etc., in logging, hardcoded, or in specs)
- [x] Provide which alerts would indicate a problem with this functionality (if applicable)

### CODE REVIEW

File: lib/saml/user.rb [hunk 1]
```diff
@@ -29,7 +29,7 @@ class User
     def initialize(saml_response)
       @saml_response = saml_response
       @saml_attributes = saml_response.attributes
-
+puts saml_attributes&.to_h
```
Comment (jholton) [inline lib/saml/user.rb:32]: debug line?

**Comment by ericbuckley** [comment]
@annaswims what about `va_eauth_uid` and `va_eauth_secid` should we be filtering those at as well?

File: lib/saml/user.rb (hunk 1 above)
Comment (annaswims) [reply lib/saml/user.rb:32]: Guilty.

**Comment by annaswims** [comment]
va_eauth_{first|middle|last}name are being filtered already because the keys include `{first|middle|last}name`

**Comment by annaswims** [comment]
@ericbuckley are va_eauth_uid and va_eauth_secid considered PII or are they just primary keys that are only useful if you have access to our database?

**Comment by jholton** [comment]
> @ericbuckley are va_eauth_uid and va_eauth_secid considered PII or are they just primary keys that are only useful if you have access to our database?

This ☝️ specific question needs to be answered, but _if_ they are "just primary keys", I think the rationale presented in [this PR](https://github.com/department-of-veterans-affairs/vets-api/pull/2888#issuecomment-474173431) applies.
(Unfortunately, I don't see "SEC ID" mentioned in the doc that is linked to in the PR comment)

**Comment by annaswims** [comment]
I know we've talked about id.me uuid and allow it to be logged and I'd consider sec_id to be the corresponding id from SSOe.  Only useful for identification purposes if someone has access to IAM data.

**Comment by ericbuckley** [comment]
> @ericbuckley are va_eauth_uid and va_eauth_secid considered PII or are they just primary keys that are only useful if you have access to our database?

@annaswims I believe the va_eauth_secid is a unique identifier for the users MVI record, the va_eauth_uid value is a little more complicated, as depending on what credentials the user used to authenticate with this will change.  When logging in with ID.me credentials, it would be the users ID.me UUID, however when logging in with other credentials it could be something else.  Its probably safest to filter that value, however @patrickvinograd could probably provide more context.

**Pull request closed without merging**